### PR TITLE
Fix openqa crash

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: oqamaint/update.py [hunk 1]
```diff
@@ -56,15 +56,14 @@ def get_max_revision(self, job):
     def settings(self, src_prj, dst_prj):
         s = self._settings.copy()
 
+        patch_id = src_prj.split(':')[-1]
```
Comment (mimi1vx) [inline oqamaint/update.py]: this is equal to `def patch_id(repo):`? In INCIDENT patch is needed a real name of patch of which is incident number only small part

File: oqamaint/openqabot.py [hunk 2]
```diff
@@ -408,7 +411,7 @@ def check_one_request(self, req):
             self.logger.error(traceback.format_exc())
             return None
 
-        return ret
+        return
```
Comment (mimi1vx) [inline oqamaint/openqabot.py]: define ret=None at beginning of function and not needed explicit return on every if branch

File: oqamaint/update.py (hunk 1 above)
Comment (coolo) [reply oqamaint/update.py]: hmm, not for openSUSE. that's what fooled me.

The actual problem is that https://download.opensuse.org/repositories/openSUSE:/Maintenance:/8006/openSUSE_Leap_15.0_Update/repodata/ does not have an updateinfo.xml - which might be a bug.

Comment (coolo) [reply oqamaint/update.py]: CC @andreasstieger

File: oqamaint/openqabot.py (hunk 2 above)
Comment (coolo) [reply oqamaint/openqabot.py]: I know, but I like it better to have the returns more local. That's why I changed it to begin with - I just forgot about one branch

File: oqamaint/update.py (hunk 1 above)
Comment (coolo) [reply oqamaint/update.py]: So I checked and you are right - my patch only works for openSUSE.

And http://download.opensuse.org/repositories/openSUSE:/Maintenance:/8027/openSUSE_Leap_15.0_Update/repodata/ also doesn't have updateinfo, but it must have had at some point.

**Comment by coolo** [comment]
29234921e34c9199fc65363a1a50f2c522eae914 is the next best thing

**Comment by coolo** [comment]
there is always a default return value in python

**Comment by mimi1vx** [comment]
@coolo but not default variable `ret`

so `return` returns `None`
but `return ret` without defined ret throws `NameError`

**Comment by coolo** [comment]
I see now - I managed to confuse myself with branches.

**Comment by coolo** [comment]
(git branches not python if/else branches - that statement wasn't clear :)

**Pull request merged**